tests/solvers/test_solvers.py

```diff
@@ -33,6 +43,17 @@ def test_scipyopt(self, rb):
         solver.fit(data=None, options={'maxiter': 20})
         assert np.abs(solver.fun_val_opt) < 1e-5
 
+    @pytest.mark.parametrize('curve_fun', [ln_gaussian_pdf, ln_gaussian_cdf, gaussian_pdf, gaussian_cdf])
```

Comment:
Can we do a few of these to catch unusual simulations? So maybe
```
@pytest.markparametrize('seed', np.arange(10))
def test_scipyopt_core_model(self, curve_fun, seed):
    np.set_seed(seed)
    ...
```

Reply:
Yes, definitely! Once I finish putting together all the tests, I'll let it run multiple times.